src/providers/image-loader.ts

```diff
@@ -67,11 +69,28 @@ export class ImageLoader {
   }
 
   private get isWKWebView(): boolean {
-    return this.platform.is('ios') && (<any>window).webkit;
+    if (this.platform.is('ios') && (<any>window).webkit && (<any>window).webkit.messageHandlers) {
+      return true;
+    }
+
+    return false;
   }
 
   private get isIonicWKWebView(): boolean {
-    return this.isWKWebView && location.host === 'localhost:8080';
+    if (this.isWKWebView) {
```

Comment:
I'd rather make this method do this:
```ts
return this.isWKWebView && (location.host === 'localhost:8080'  || (<any>window).LiveReload);
```

Then we can remove `declare var LiveReload;`